src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -345,6 +390,131 @@ private static ConfigDef createConfigDef() {
         Width.MEDIUM,
         CONNECTION_URI_DISPLAY);
 
+    configDef.define(
```

Comment:
What's the rationale for all the changes to configDef?

Reply:
Purely organizational